File: modules/spring-core/testdata/java/inspection/suppresses/testConfiguration/src/MockTest.java
import org.springframework.boot.test.mock.mockito.MockBean;
import org.springframework.boot.test.mock.mockito.SpyBean;
import org.springframework.stereotype.Component;

@Component
public class MockTest {
    @MockBean
    DemoService demoServiceMock;
    @SpyBean
    DemoService demoServiceSpy;
}

@Component
class DemoService {}

File: modules/spring-core/testdata/kotlin/inspection/suppresses/testConfiguration/src/MockTest.kt
import org.springframework.boot.test.mock.mockito.MockBean;
import org.springframework.boot.test.mock.mockito.SpyBean;
import org.springframework.stereotype.Component;

@Component
public class MockTest {
    @MockBean
    lateinit var demoServiceMock: DemoService

    @SpyBean
    lateinit var demoServiceSpy: DemoService
}

@Component
class DemoService {}